code/game/machinery/syndicatebomb.dm

```diff
@@ -434,10 +438,6 @@
 
 	playsound(loc, 'sound/effects/bamf.ogg', 75, TRUE, 5)
 
-	if(loc && istype(loc, /obj/machinery/syndicatebomb/))
```

Comment:
What's going on here ? Surely when it's done core should be gone ?

Reply:
I'm currently running on the principal that the bomb core is basically just a really, really big chemical grenade and the casing is the wiring to make it go off/anchor it to the floor. I'm leaving it up to the explosion it creates to destroy the core and casing. If there's no explosion I don't see a reason that either should be destroyed.